catalyst_voices/apps/voices/lib/widgets/document_builder/document_checkbox_builder_widget.dart

```diff
@@ -0,0 +1,123 @@
+import 'package:catalyst_voices/widgets/document_builder/document_property_footer.dart';
+import 'package:catalyst_voices/widgets/document_builder/document_property_topbar.dart';
+import 'package:catalyst_voices/widgets/widgets.dart';
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+import 'package:flutter/material.dart';
+
+class DocumentCheckboxBuilderWidget extends StatefulWidget {
+  final DocumentProperty property;
```

Comment:
Imo this widget should not now about `DocumentProperty`. Just declare fields that you need